app/help/en/gateways/ioxbank.md

```diff
@@ -0,0 +1,42 @@
+# ioxbank Instant Swap Service
```

Comment:
Please remove the notion of "instant" as long as your service does not operate within one block.

Reply:
Instant means once blocks are confirmed and it's the correct notion.

Reply:
I disagree that this reflects an Instant Swap, I think it is misleading.

Reply:
Perhaps together "Instant Swap" would be misleading to "Atomic Swap"; I agree with this.

I can change to "Instant Gateway Service", do you agree with this? I still believe "instant" is relevant as the transaction happens as instant as possible and time is only required for proofing the Blocks confirmation which take seconds.

Reply:
Instant Gateway Service is acceptable, please do clarify in the text what instant means.

Reply:
> Instant Gateway Service is acceptable to me, please do clarify in the text what instant means.

done: https://github.com/bitshares/bitshares-ui/blob/f63df4bc759a408520c4a3e85fcad8a2934e8cad/app/help/en/gateways/ioxbank.md